pygameMenu/menu.py

```diff
@@ -276,17 +301,36 @@ def __init__(self,
                         (self._posx + self._width, self._posy + self._height),
                         (self._posx, self._posy + self._height)
                         ]
-        self._draw_regionx = draw_region_x
-        self._draw_regiony = draw_region_y
+        self._draw_regionx = draw_region_x  # type: int
+        self._draw_regiony = draw_region_y  # type: int
 
         # Option position
-        self._opt_posx = int(self._width * (self._draw_regionx / 100.0)) + self._posx
         self._opt_posy = int(self._height * (self._draw_regiony / 100.0)) + self._posy
-        self._widget_align = widget_alignment
+
+        # Columns support
+        if columns > 1:
+            if column_weights is None:
+                column_weights = tuple(1 for _ in range(columns))
+            s = float(sum(column_weights[:columns]))
+            cumulative = 0
+            self._column_posx = []
+            for i in range(columns):
+                w = column_weights[i] / s
+                self._column_posx.append(
+                    int(self._width * (self._draw_regionx / 100.0 + (cumulative + 0.5 * w - 0.5)) + self._posx))
+                cumulative += column_weights[i] / s
+            self._opt_posx = self._column_posx[0]
+            self._column_widths = tuple(int(self._width * column_weights[i] / s) for i in range(columns))
+        else:
+            self._opt_posx = int(self._width * (self._draw_regionx / 100.0) + self._posx)
+        self._column_spacing = int(self._width / columns)  # type: int
+        self._columns = columns  # type: int
+        self._force_fit_text = force_fit_text  # type: bool

```

Comment:
Note that the types of menu options are already defined in the docstring. What is the use of redefining them near the attribute?
This can lead to an inconsistency if the type changes and one of the two places is forgotten to be updated.

Reply:
mmm right, I'll fix them